build/scheduling-node-operator/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13 as builder
```

Comment:
in the other Dockerfiles we updated to 1.14

Reply:
updated, thanks!